hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -164,7 +165,13 @@ private void syncHoodieTable(String tableName, boolean useRealtimeInputFormat) {
     LOG.info("Storage partitions scan complete. Found " + writtenPartitionsSince.size());
     // Sync the partitions if needed
     syncPartitions(tableName, writtenPartitionsSince);
-
+    // Sync the table properties if need
+    if (cfg.tableProperties != null) {
+      Map<String, String> tableProperties = ConfigUtils.toMap(cfg.tableProperties);
+      hoodieHiveClient.updateTableProperties(tableName, tableProperties);
+      LOG.info("Sync table properties for " + tableName + ", table properties is: "
+          + cfg.tableProperties);
+    }
```

Comment:
Can't we sync this while creating the table itself, like you are doing for serde properties ?

Reply:
Well, the `tableProperties` may change if the schema has changed. So we need to update the table properties  by a separate interface.

Reply:
Yeah, +1 for this.